.taskcluster.yml

```diff
@@ -82,7 +82,9 @@ tasks:
                           then: '3'
                           else: '1'
                   in:
-                      taskId: '${ownTaskId}'
+                      taskId:
+                          $if: 'tasks_for != "action"'
```

Comment:
Question: Oh, does `null` work? If so, I could simplify the logic I wrote in fenix 🙂:

https://github.com/mozilla-mobile/fenix/blob/3d387e2dde9bf2f0265c9467a1d9e04c683c3144/.taskcluster.yml#L117-L120

Reply:
It's working for me, for xpi-manifest: https://github.com/mozilla-extensions/xpi-manifest/blob/master/.taskcluster.yml#L91-L93

Reply:
Awesome!